exercises/D-logical-operators/exercise2.js

```diff
@@ -5,8 +5,33 @@
   Update the code so that you get the expected result.
 */
 
-function isNegative() {}
+function isNegative(num) {
+  if (num > 0) {
+    return false;
+  }
+  return true;
+}
 
+function isBetween5and10(num) {
+  if (num >= 5 && num <= 10) {
+    return true;
+  }
+  return false;
+}
+
+function isShortName(name) {
+  if (name.length < 7) {
+    return true;
+  }
+  return false;
+}
+
```

Comment:
plz. try to think about how you can make more concise these functions
hint:
let name = "Salva"
let test = name.length < 7
console.log(typeof test)